ui/pages/confirmation/confirmation.js

```diff
@@ -176,9 +176,10 @@ export default function ConfirmationPage({
   };
 
   ///: BEGIN:ONLY_INCLUDE_IN(flask)
-  const {
-    manifest: { proposedName },
-  } = useSelector((state) => getSnap(state, pendingConfirmation?.origin));
+  const snap = useSelector((state) => getSnap(state, pendingConfirmation?.origin));
+
+  // When pendingConfirmation is undefined, this will also be undefined
```

Comment:
Should we update lines 233, 236, and 306 as well since pendingConfirmation could be undefined?

Reply:
Good catch! I have moved these to be below the early return, that should alleviate any potential problems.